release-pipeline.yml

```diff
@@ -0,0 +1,1737 @@
+variables:
+  AZCOPY_GOLANG_VERSION: '1.21.8'
+  AZCOPY_GOLANG_VERSION_COVERAGE: '1.21.8'
+  AZCOPY_VERSION: '0.0.0'
+  MUTEX_URL: https://ciazcopyv10regular.blob.core.windows.net/distributedmutex/lock
```

Comment:
Couldn't see where this variable is used. If this is not required, please remove it.

Reply:
removed